modules/ducktests/tests/ignitetest/services/utils/config_template.py

```diff
@@ -43,7 +44,10 @@ def render(self, **kwargs):
         Render configuration.
         """
         kwargs.update(self.default_params)
-        res = self.template.render(**kwargs)
+        unfiltered = self.template.render(**kwargs)
+
+        res = '\n'.join(list(filter(lambda line: re.search(r'\S', line), unfiltered.split('\n'))))
```

Comment:
join accepts iterator, there is no need to create list

Reply:
thanks, done